src/sentry/static/sentry/app/components/versionHoverCard.tsx

```diff
@@ -45,51 +57,6 @@ class VersionHoverCard extends React.Component<Props, State> {
     release: null,
   };
 
-  componentDidMount() {
-    this.fetchData();
-  }
-
-  async fetchData() {
-    const {api, orgSlug, projectSlug, releaseVersion} = this.props;
-
-    // releases
-    const releasePath = `/projects/${orgSlug}/${projectSlug}/releases/${encodeURIComponent(
-      releaseVersion
-    )}/`;
-    const releaseRequest = api.requestPromise(releasePath, {
-      method: 'GET',
-    });
-
-    // repos
-    const repoRequest = api.requestPromise(`/organizations/${orgSlug}/repos/`, {
-      method: 'GET',
-    });
-
-    //deploys
-    const deployPath = `/organizations/${orgSlug}/releases/${encodeURIComponent(
-      releaseVersion
-    )}/deploys/`;
-    const deployRequest = api.requestPromise(deployPath, {
-      method: 'GET',
-    });
-
-    try {
-      const [release, repos, deploys] = await Promise.all([
-        releaseRequest,
-        repoRequest,
-        deployRequest,
-      ]);
-      this.setState({
```

Comment:
@leedongwei the `state` of this component can now be simplified since they're now prop driven.

Reply:
Oh yea, good catch 🙏